### DISCUSSION TITLE
More utilization for the toolbelt slot

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#4734] [Ideas and suggestions]
### More items that can fit in toolbelt slot would be nice, for example 
**Backpacks** that reduce your movement speed but have increased inventory capacity, 
**Transmitter** backpacks that extend communication range to 500m, 
**Large oxygen tanks** akin to scuba divers, 
**Tactical vest** that have higher max stack capacity for ammunition,
 **Fanny packs/basic belt** (3 slot toobelts, while making toolbelts slightly more expensive to craft), 
**Back _Racks_** that can hold medium/large items to transport railgun shells for example (with major speed penalty of course) 
And role play elements like a **guitar sling-on** (so you can bring your guitar along) and **clown drum kit**. 

Further more, additional variety for toolbelts like a medic bag, mechanic's sling on, decorated clown bag, would give a more immersive/believable interactions with **"job related toolbelts"**

**Comment by Saevon** [comment]
**Oxygen Tank belt**:  holds 3 oxygen tanks, automatically uses them for a dive suit / dive helmet / tool you're wearing.  (Uses belt before any other supply)
**Power Belt**: Holds 3 battery cells which are used for your equipment automatically
**Wire belt**:  ALL THE WIRES! holds up to 6 wires, which can now stack x3 more (compared to its stacks in the hopefully coming updated)
